bims/views/map.py

```diff
@@ -20,4 +21,7 @@ def get_context_data(self, **kwargs):
         context['bing_map_key'] = get_key('BING_MAP_KEY')
         context['map_tiler_key'] = get_key('MAP_TILER_KEY')
         context['geocontext_url'] = get_key('GEOCONTEXT_URL')
+        context['collectors'] = \
```

Comment:
I think it's better to request the collectors with api and backbone fetch. Avoiding long load of the map page if we have thousands collectors.

Reply:
hi @ann26 
i agree with dimas. Instead put it on context, maybe we could load it through api and add throbber on UI. We need to page to be loaded fast.

So, when page load -> call api to get collectors and render it. Add throbber on the element.

Reply:
done